content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md

```diff
@@ -0,0 +1,56 @@
++++
+date = "2017-08-11"
+title = "Quarterly update: Continuous improvement, more tools to help you use cloud.gov"
++++
+
+Every quarter we update you with the important news about new features and updates to the cloud.gov platform. In this edition of the newsletter we have updates about the cloud.gov dashboard and other features for our users as well as stories from our customers, and links to recent presentations about cloud.gov.
+
+### What’s new
+
+#### Use the dashboard to invite new users to your team
+
+Managing users in your organization and granting them the permissions they need should be a straightforward process so your team can get to work building your app on cloud.gov. To make it easier for you, we built new features for inviting new users and giving them roles within your organization using the dashboard. With these changes, even the least technical people on your team can invite new folks and manage their role without involving cloud.gov support.
+
+<img src="/img/invite-users.png" alt="The user invitation screen with notification" width="624" height="272" />
+
+Being able to manage your users from the dashboard will also allow you to have finer control over who has access to different parts of your system without having to involve your technical staff. Our users requested this functionality, so if you get a chance to use it please let us know how it’s working for you. Watch for more developments like this in the future.
+
+#### Continuous improvement
+
+Staying compliant with federal laws is crucial to us and to you, but compliant simply isn’t good enough, or fast enough, for us. That’s why [cloud.gov is designed and operated to routinely provide apps the latest security updates](https://www.google.com/url?q=https://cloud.gov/overview/overview/cloudgov-benefits/&sa=D&ust=1501263385089000&usg=AFQjCNGRQGmoDF2V5KRSRg1JaAFp_yMYYg) within a day of their release. When there’s a major vulnerability in a component managed by cloud.gov, our team makes these updates across the entire platform, so you don’t have to spend time doing it. For updates to the buildpack, operating system, or [any other component we manage](https://cloud.gov/overview/technology/responsibilities/), all that’s typically required is a [restage of your apps](https://cloud.gov/docs/getting-started/app-maintenance/#restaging-your-app), even as they scale and grow, making security straightforward and easy for you.
```

Comment:
"cloud.gov is designed and operated to routinely provide apps the latest security updates" ended up with an odd Google URL link.